CONTRIBUTE.md

```diff
@@ -0,0 +1,154 @@
+# Contributing to PlasmaPy
+
+There are numerous ways to contribute to PlasmaPy, including by
+providing code and documentation, suggesting and discussing ideas,
+submitting issues and bug reports, and engaging the broader plasma
+physics community.  
+
+## Sharing ideas
+
+There are several methods of communication that are being used in the
+early stages of PlasmaPy development:
+
+* [Signing up for the PlasmaPy email
+  list](https://groups.google.com/forum/#!forum/plasmapy) will allow
+  you to participate in broader discussions and keep up with .
+
+* The [PlasmaPy repository on
+  GitHub](https://github.com/PlasmaPy/plasmapy) is the best place to
+  [submit issues](https://github.com/PlasmaPy/plasmapy/issues) and
+  review [pull requests](https://github.com/PlasmaPy/plasmapy/pulls).
+
+* The [PlasmaPy HipChat group](https://plasmapy.hipchat.com/chat) is a
+  great place to have informal conversations and share ideas.  Before
+  participating it is necessary to join the PlasmaPy team on HipChat
+  (note: if you have another HipChat account, then you must use a
+  different email address for this one).
+
+* We have regularly scheduled telecons which are announced on the
+  email list.
+
+## Contributing code or documentation to PlasmaPy
+
+### Preliminaries
+
+Before contributing to the PlasmaPy code base, one must [**join
+GitHub**](https://github.com/join?source=header-home).  A free account
+will suffice for you to have unlimited public repositories.  If you
+are new to [git](https://git-scm.com/), helpful resources include
+documentation on [git
+basics](https://git-scm.com/book/en/v2/Getting-Started-Git-Basics) and
+an [interactive git
+tutorial](https://try.github.io/levels/1/challenges/1).  You must also
+[install
+git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+locally on your computer.
+
+### Forking and cloning PlasmaPy
+
+After creating your GitHub account, go to the [main
+repository](https://github.com/PlasmaPy/PlasmaPy) and **fork a copy of
+PlasmaPy to your account**.
+
+Next you must **clone your fork to your computer**.  Go to the
+directory that will host your PlasmaPy directory, and run one of the
+following commands (after changing *your-username* to your username).
+If you would like to use HTTPS, then run:
+```
+git clone https://github.com/*your-username*/PlasmaPy.git
+```
+If you have previously [set up an SSH
+key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/), then you may run:
+```
+git clone git@github.com:*your-username*/PlasmaPy.git
+```
```

Comment:
I'd add here that https is the default (and easier to set up) but that ssh is more secure, just to give people a way of deciding between the two if they're not familiar with git.